recipes/openmeeg/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -DBLA_VENDOR=OpenBLAS -DENABLE_PYTHON=OFF -DCMAKE_BUILD_TYPE=RELEASE -DBUILD_DOCUMENTATION=OFF %SRC_DIR%
```

Comment:
Would wrap this to make it easier to read like [so]( https://github.com/conda-forge/gflags-feedstock/blob/master/recipe/bld.bat ).